libs/design-system/brandmark/src/lib/brandmark.component.ts

```diff
@@ -0,0 +1,27 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, computed, input } from '@angular/core';
+import { AssetUrlPipe } from '@hra-ui/cdk/app-href';
+
+/**
+ * HRA brandmark component
+ */
+@Component({
+  selector: 'hra-brandmark',
+  standalone: true,
+  imports: [CommonModule, AssetUrlPipe],
+  templateUrl: './brandmark.component.html',
+  styleUrl: './brandmark.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BrandmarkComponent {
+  /** Brandmark color variant */
+  readonly color = input<string>('default');
+
+  /** Brandmark size to use */
+  readonly small = input<boolean>(false);
```

Comment:
Since the brandmark is an svg it should automatically scale to whatever size we need so this input is not needed.